contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2151,3 +2151,45 @@ SELECT
   CAST('' as NVARCHAR(60)) AS non_transacted_access_desc
 WHERE FALSE;
 GRANT SELECT ON sys.database_filestream_options TO PUBLIC;
+
+CREATE OR REPLACE VIEW sys.xml_indexes
+AS
+SELECT
+    CAST(idx.object_id AS INT) AS object_id
+  , CAST(idx.name AS sys.sysname) AS name
+  , CAST(idx.index_id AS INT)  AS index_id
+  , CAST(idx.type AS sys.tinyint) AS type
+  , CAST(idx.type_desc AS sys.nvarchar(60)) AS type_desc
+  , CAST(idx.is_unique AS sys.bit) AS is_unique
+  , CAST(idx.data_space_id AS int) AS data_space_id
+  , CAST(idx.ignore_dup_key AS sys.bit) AS ignore_dup_key
+  , CAST(idx.is_primary_key AS sys.bit) AS is_primary_key
+  , CAST(idx.is_unique_constraint AS sys.bit) AS is_unique_constraint
+  , CAST(idx.fill_factor AS sys.tinyint) AS fill_factor
+  , CAST(idx.is_padded AS sys.bit) AS is_padded
+  , CAST(idx.is_disabled AS sys.bit) AS is_disabled
+  , CAST(idx.is_hypothetical AS sys.bit) AS is_hypothetical
+  , CAST(idx.allow_row_locks AS sys.bit) AS allow_row_locks
+  , CAST(idx.allow_page_locks AS sys.bit) AS allow_page_locks
+  , CAST(idx.has_filter AS sys.bit) AS has_filter
+  , CAST(idx.filter_definition AS sys.nvarchar) AS filter_definition
```

Comment:
shouldn't we cast this to nvarchar(max)?

Reply:
Since it's in `plpgsql` it will produce a syntax error. Should we do `sys.nvarchar(4000)` instead?

Reply:
Chatted offline, and looks like the atttypmod value for this column (from pg_attribute view) is -1 which should be okay to use

Reply:
typmod = -1 would be considered as typmod = 30 for sys.nvarchar. check https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/contrib/babelfishpg_tsql/src/pltsql_utils.c#L93. So let's use sys.nvarchar(4000) for the time being.

Reply:
Sounds good!